src/Modules/Main.lua

```diff
@@ -66,9 +66,7 @@ function main:Init()
 	end
 	
 	self.tree = { }
-	for _, treeVersion in ipairs(treeVersionList) do
-		self.tree[treeVersion] = new("PassiveTree", treeVersion)
-	end
+	self.tree[latestTreeVersion] = new("PassiveTree", latestTreeVersion)
```

Comment:
```suggestion
	self:LoadTree(latestTreeVersion)
```
Maybe just use the function